frontend/src/components/ContentBox.tsx

```diff
@@ -15,8 +25,20 @@ const ContentBox = ({
   if (!children) {
     return null;
   }
+
+  // 224px represents the current height of the <Footer>. If more items are
+  // added in it, its height may increase.
+  const minHeight =
+    window.innerHeight - topBarHeight - contentHeaderHeight - 224;
```

Comment:
Perfect :ok_hand: 

My initial proposition was to always display it at the bottom of the screen to avoid displaying a supposedly useless scroll bar.

Reply:
Decided with the team, the footer will be always out of view.